lang/src/test/kotlin/org/partiql/lang/eval/builtins/windowFunctions/WindowFunctionTests.kt

```diff
@@ -303,10 +303,10 @@ class WindowFunctionTests : EvaluatorTestBase() {
                     GROUP BY EXTRACT(MONTH FROM sp."date") as month, sp.ticker as ticker GROUP AS g
                 """,
                 expectedResult = """<<
-                    { 'current_month': 9, 'ticker': 'AMZN', 'current_month_average': 113.00, 'next_month_avg': 118.485},
-                    { 'current_month': 10, 'ticker': 'AMZN', 'current_month_average': 118.485, 'next_month_avg': NULL},
-                    { 'current_month': 9, 'ticker': 'GOOG', 'current_month_average': 96.15, 'next_month_avg': 100.17},
-                    { 'current_month': 10, 'ticker': 'GOOG', 'current_month_average': 100.17, 'next_month_avg': NULL}
+                    { 'current_month': 9., 'ticker': `AMZN`, 'current_month_average': 113.00, 'next_month_avg': 118.485},
```

Comment:
The `EXTRACT(MONTH FROM sp."date")` also returns decimal in the current implementation of PartiQL

Reply:
I don't know if we want to change the evaluation logic in `EXTRACT`, so it outputs an integer for month